server/logging/logging.go

```diff
@@ -74,6 +74,7 @@ func New(config config.KolideConfig, logger log.Logger) (*OsqueryLogger, error)
 		status, err = NewPubSubLogWriter(
 			config.PubSub.Project,
 			config.PubSub.StatusTopic,
+			false,
```

Comment:
Is it intended not to set these for status logs?

Reply:
Yes - the status logs have a different set of fields, and I think in general this feature is less useful for them.